WalletWasabi/Tor/Control/Messages/CircuitStatus/CircuitInfo.cs

```diff
@@ -46,9 +46,9 @@ public CircuitInfo(string circuitID, CircStatus circStatus)
 	public static CircuitInfo ParseLine(string line)
 	{
 		(string circuitId, string remainder1) = Tokenizer.ReadUntilSeparator(line);
-		(string circuitStatus, string remainder2) = Tokenizer.ReadUntilSeparator(remainder1);
+		(string circuitStatusString, string remainder2) = Tokenizer.ReadUntilSeparator(remainder1);
```

Comment:
I have changed this to not be conflicted with the variable on L51.
If anyone has a better alternative I would like to change it.